java-squid/src/main/java/org/sonar/java/se/symbolicvalues/NotEqualRelation.java

```diff
@@ -31,12 +31,18 @@
 
   @Override
   protected BinaryRelation symmetric() {
-    return new NotEqualRelation(rightOp, leftOp);
+    if(symmetric == null) {
+      symmetric = binaryRelation(RelationalSymbolicValue.Kind.NOT_EQUAL, rightOp, leftOp);
+    }
+    return symmetric;
   }
 
   @Override
   public BinaryRelation inverse() {
-    return new EqualRelation(leftOp, rightOp);
+    if(inverse == null) {
+      inverse = BinaryRelation.binaryRelation(RelationalSymbolicValue.Kind.EQUAL, leftOp, rightOp);;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)